crates/sui-sdk/src/apis.rs

```diff
@@ -329,6 +641,25 @@ impl CoinReadApi {
         Ok(self.api.http.get_all_coins(owner, cursor, limit).await?)
     }
 
+    /// Return the coins for the provided address as a stream.
```

Comment:
Worth explaining how this compares to `get_all_coins` -- i.e. why would someone want to use this vs `get_coins` or `get_all_coins` directly.